### PR TITLE
Put keynote slides

### PR DESCRIPTION
**Pull request by Maliaotw**
## Types of changes

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
put keynote slides url to file

### CODE REVIEW

**Review by uranusjr — APPROVED** [review]

**Comment by yychen** [comment]
Wait wait... @Maliaotw 你只是把 json 加了這個欄位但是前端 template 好像不會秀阿 orz

**Comment by Maliaotw** [comment]
修正了， 請在幫我看看

File: src/templates/pycontw-2020/contents/_default/conference/keynotes.html [hunk 1]
```diff
@@ -20,66 +20,72 @@
...
+							{{ data.session.description|default:placeholder_description|linebreaks }}
+						</article>
+						{% if data.slido %}
+							<article data-target="tabbing.pane">
+								<iframe class="slido" src="{{ data.slido }}"></iframe>
+							</article>
+						{% endif %}
+						<article data-target="tabbing.pane">
+							<a href="{{ data.slides }}">{{ data.slides }}</a>
+						</article>
```
Comment (yychen) [inline src/templates/pycontw-2020/contents/_default/conference/keynotes.html:82]: 你這樣的話，第一是那一整個 tab 會很空，第二個是沒有投影片的人這邊就會點下去是空白的...

我在想，你有沒有辦法把他簡單的做，加在 "演講" 的那個 tab 的內容裡，但是變成很像 social links 那樣，如果有投影片，就放在最下面，例如
```html
  {% if data.slides %}
	 <div class="slides">
		  Slides: <a href="{{ data.slides }}" target="_blank" rel="noopener">{{ data.slides}}</a>
	 </div>
  {% endif %}
```

**Review by yychen — CHANGES_REQUESTED** [review]
另外，就是你的 IDE 好像都會把整個縮排做處理，其實這樣就會很難看得出你到底實際上改的是哪些地方 orz

**Comment by tai271828** [comment]
@Maliaotw is this merge request ready to review? If no, you may convert this pr to "draft" (this is a github feature). If yes, please signal us.

**Comment by mattwang44** [comment]
@tai271828 Yes, this PR is done and is currently under review by me. However, since this PR (& also #782) is a part of the 2020 website, I personally wish Tom to be the final reviewer. It's my fault to make it being stale for so long since Tom & I haven't kicked-off the handover tasks until tomorrow (we'll have a meeting in person and go through the checklist for closing the 2020 website). So, please give us more time, and we'll close this PR soon 😅.

**Comment by tai271828** [comment]
@mattwang44 no problem! I asked to make sure this PR is in people's radar, so we won't waste @Maliaotw 's contribution : )

**Review by yychen — APPROVED** [review]
ㄜ! 這跟我預期的不太一樣，但這樣子也是可以的啦 orz.

**Pull request merged**